CSCD488-Winter2024/senior-project-the-a-team
# Intro pages

to test:

create a new account and walk through the tour.

test skipping tour at various points in the tour to ensure that functionality is working everywhere.

you can change the boolean in your account back to 'false' and refresh the app to run the tour again.

Feel free to give me feedback for improvements of the text.
